src/main/java/com/brandwatch/internship/fetchandstoretweets/services/FetchAndStoreService.java

```diff
@@ -0,0 +1,67 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.MentionsRepository;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.social.twitter.api.SearchResults;
+import org.springframework.social.twitter.api.Tweet;
+import org.springframework.social.twitter.api.Twitter;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Service
+public class FetchAndStoreService {
+
+    private Twitter twitter;
+
+    private MentionsRepository mentionsRepository;
+    private QueryRepository queryRepository;
+
+    private int counter;
+    private String[] searchStrings = {"#maven", "#spring", "#intellij_idea", "#kafka"};
+
+    @Autowired
+    public FetchAndStoreService(Twitter twitter, MentionsRepository mentionsRepository,
+                                QueryRepository queryRepository) {
+
+        this.twitter = twitter;
+        this.mentionsRepository = mentionsRepository;
+        this.queryRepository = queryRepository;
+
+        this.mentionsRepository.deleteAll();
+        this.queryRepository.deleteAll();
+
+    }
+
+    @Scheduled(fixedRate = 5000)
```

Comment:
It would be nicer to extract scheduled functionality to a separate class, call it `FetchTweetsJob`.This way you have two classes withe better separation of concerns - `Job` is responsible for scheduled communication with twitter, and `Service` responsible for matching tweets to queries and saving them into database.